MediaWikiKit/MediaWikiKit/MWKSection.m

```diff
@@ -106,6 +106,10 @@ - (MWKTitle*)sourceTitle {
     }
 }
 
+- (BOOL)hasTextData {
+    return [self.article.dataStore htmlFileExistsForSectionId:self.sectionId title:self.article.title];
+}
```

Comment:
Note: we want this to return YES even if the "Section.html" file is empty - this is because some sections *do* have zero length text strings - such as sections which have immediate sub-sections. If we didn't do this articles having such sections would *never* appear to be cached. I added a test to cover this :)